packages/story-editor/src/components/form/advancedDropDown/list/styled.js

```diff
@@ -82,7 +82,7 @@ export const Option = styled.li.attrs(({ fontFamily }) => ({
   }
 `;
 
-export const Selected = styled(Checkmark)`
+export const Selected = styled(Icons.Checkmark)`
```

Comment:
Here let's use the `CheckmarkSmall` icon, it matches figma better. 

We're going to have to fudge the placement a little bit because this dropdown's spacing is applied above the checkmark so the space it's being given doesn't start at the edge of the menu, it starts at the edge of the list item. 
![Screen Shot 2021-08-16 at 4 07 48 PM](https://user-images.githubusercontent.com/10720454/129640002-a68f6a74-f1fb-4739-8a68-42bc1fd47c97.png)


Since we're directly interacting with the SVG by styling it, we're going to want to reinforce the width (32px) and set the height to auto so it can adjust itself accordingly. 

Now we just have to play with the margin a smidge to figure out where this goes. If we inspect the list items we can see they are getting `padding: 8px 16px`, we want the icon to line up with the edge of the list item so we want to give some negative margin. 

It'll end up being: 
```
export const Selected = styled(Icons.CheckmarkSmall)`
  display: inline-block;
  width: 32px;
  height: auto;
  margin-left: -8px;
  margin-bottom: -10px;
  color: ${({ theme }) => theme.colors.fg.primary};
`;
```